financial-connections/src/main/java/com/stripe/android/financialconnections/ui/theme/Theme.kt

```diff
@@ -56,6 +56,14 @@ private val Colors = FinancialConnectionsColors(
     borderBrand = Color(0xFF675DFF),
 )
 
+internal val InstantDebitsColors = Colors.copy(
+    textBrand = Color(0XFF00A355),
+    iconBrand = Color(0XFF00D66F),
+    buttonPrimary = Color(0XFF00D66F),
+    buttonPrimaryPressed = Color(0XFF00D66F),
+    borderBrand = Color(0XFF00D66F),
+)
```

Comment:
for now just updating the "accent" color to Link. Just used on the pre-browser spinner.